CHANGELOG.md

```diff
@@ -17,12 +17,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Updated
 
-## [v6.8.1] - 2025-12-15
+## [v6.8.2] - 2025-12-19
```

Comment:
Hi @johnmichaelmurner This pr looks really good. I will do a better review. Can you move your changes in the changelog to the Unreleased section and not change the version number? Thank you.